QRHunter/app/src/main/java/com/example/qrhunter/ui/leaderboard/LeaderboardFragment.java

```diff
@@ -4,26 +4,70 @@
 
 import androidx.fragment.app.Fragment;
 import androidx.lifecycle.ViewModelProvider;
+import androidx.navigation.Navigation;
+import androidx.recyclerview.widget.RecyclerView;
 
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
+import android.widget.ListView;
+import android.widget.SearchView;
+import android.widget.Toast;
 
 import com.example.qrhunter.R;
+import com.example.qrhunter.data.model.Player;
+import com.example.qrhunter.data.repository.UserRepository;
 import com.example.qrhunter.databinding.FragmentLeaderboardBinding;
 
+import java.util.ArrayList;
+
 public class LeaderboardFragment extends Fragment {
     private FragmentLeaderboardBinding binding;
 
+    private LeaderboardViewModel leaderboardViewModel;
+
+    private ListView recyclerView;
```

Comment:
Curious why you're calling it recyclerView if you're using ListView?

Reply:
oh i was planning to use a recyclerview at first but changed it later, ill change it